x/challenge/keeper/msg_server_submit.go

```diff
@@ -24,8 +26,11 @@ func (k msgServer) Submit(goCtx context.Context, msg *types.MsgSubmit) (*types.M
 		return nil, types.ErrUnknownBucketObject
 	}
 	sp := k.StorageKeeper.MustGetPrimarySPForBucket(ctx, bucketInfo)
+
 	if sp.Status != sptypes.STATUS_IN_SERVICE && sp.Status != sptypes.STATUS_GRACEFUL_EXITING {
-		return nil, types.ErrInvalidSpStatus
+		if !ctx.IsUpgraded(upgradetypes.Hulunbeier) || sp.Status != sptypes.STATUS_FORCED_EXITING {
```

Comment:
`STATUS_FORCED_EXITING` is comes together `Hulunbeier` fork
maybe, we can just check the status.

Reply:
fixed